packages/main/src/plugin/updater.ts

```diff
@@ -403,15 +404,18 @@ export class Updater {
       .checkForUpdates()
       .then(result => {
         this.#updateCheckResult = result ?? undefined;
-        this.#nextVersion = this.getFormattedVersion(this.#updateCheckResult?.updateInfo);
+        this.#nextVersion =
+          compare(app.getVersion(), this.#updateCheckResult?.updateInfo.version ?? '') === -1
+            ? this.getFormattedVersion(this.#updateCheckResult?.updateInfo)
+            : undefined;
```

Comment:
Suggestion (not blocking): A bit compact, might be a bit hard to understand, might be nice to use if else format to ease the reading